.github/workflows/publish-docusaurus.yaml

```diff
@@ -0,0 +1,115 @@
+          
+          mkdir -p ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Operation\ View/05_Migration
+          cp ./docs/migration/* ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Operation\ View/05_Migration/
+          
+          mkdir -p ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Development\ View
+          cp ./docs/README.md ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Development\ View/00_tractusx_edc.md
+          cp ./core/edr-core/README.md ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Development\ View/06_edr_core.md
+          cp ./docs/development/Release.md ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Development\ View/07_release.md
+          
+          mkdir -p ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Development\ View/03_EDC\ Controlplane
+          rsync -a --prune-empty-dirs --include '*/' --exclude 'build' --include '*.md' --include '*.png' --include '*.puml' --exclude '*' ./edc-controlplane/* ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Development\ View/03_EDC\ Controlplane
+          
+          mkdir -p ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Development\ View/04_EDC\ Dataplane
+          rsync -a --prune-empty-dirs --include '*/' --exclude 'build' --include '*.md' --include '*.png' --include '*.puml' --exclude '*' ./edc-dataplane/* ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Development\ View/04_EDC\ Dataplane
+          
+          mkdir -p ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Development\ View/05_EDC\ Extensions
+          rsync -a --prune-empty-dirs --include '*/' --exclude 'build' --include '*.md' --include '*.png' --include '*.puml' --exclude '*' ./edc-extensions/* ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Development\ View/05_EDC\ Extensions
+
+          mkdir -p ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Documentation
+          cp ./docs/development/coding-principles.md ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Documentation/02_coding_principles.md
+          cp ./pr_etiquette.md ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Documentation/03_pr_etiquette.md
+          cp ./styleguide.md ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Documentation/04_styleguide.md
+          mkdir -p ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Documentation/resources
+          cp ./resources/tx-checkstyle-config.xml ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Documentation/resources
+          cp ./SECURITY.md ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Documentation/05_security.md
+          mkdir -p ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Documentation/resources
+          cp ./resources/save_actions_scr.png ${{ runner.temp }}/tractusx-edc/Connector\ Kit/Documentation/resources/
+          
+          swaggerhub api:get ${{ env.SWAGGERHUB_USER }}/tractusx-edc/${{ env.DOWNSTREAM_VERSION }} > ${{ runner.temp }}/tractusx-edc/tractusx-edc-${{ env.DOWNSTREAM_VERSION }}.yaml
```

Comment:
generally, I recommend using `curl` to download the yaml, because using the Swaggerhub CLI fails when downloading from other APIs (e.g. `eclipse-edc`). The logged-in user / token is not valid then. Granted, that's not the case here, but using plain `curl` also avoids having to install the Swaggerhub CLI, which saves some runner execution time.